modules/test/playwright/tests/users-admin-web/organizationsItemSelector.spec.ts

```diff
@@ -0,0 +1,86 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {dataApiHelpersTest} from '../../fixtures/dataApiHelpersTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {usersAndOrganizationsPagesTest} from '../../fixtures/usersAndOrganizationsPagesTest';
+import {getRandomInt} from '../../utils/getRandomInt';
+
+export const test = mergeTests(
+	dataApiHelpersTest,
+	loginTest(),
+	usersAndOrganizationsPagesTest
+);
+
+test('LPD-24824 User only sees the organizations they have permission to view', async ({
+	apiHelpers,
+	context,
+	page,
+	usersAndOrganizationsPage,
+}) => {
+	const organization1 = await apiHelpers.headlessAdminUser.postOrganization({
+		name: 'Organization' + getRandomInt(),
+	});
+	const organization2 = await apiHelpers.headlessAdminUser.postOrganization({
+		name: 'Organization' + getRandomInt(),
+	});
+	const user = await apiHelpers.headlessAdminUser.postUserAccount();
+	await apiHelpers.headlessAdminUser.assignUserToOrganizationByEmailAddress(
+		organization1.id,
+		user.emailAddress
+	);
+	const organizationOwnerRole =
+		await apiHelpers.headlessAdminUser.getRoleByExternalReferenceCode(
+			'Organization Owner'
+		);
+	await apiHelpers.headlessAdminUser.assignUserToOrganizationRole(
+		organizationOwnerRole.id,
+		user.id,
+		organization1.id
+	);
+
+	await usersAndOrganizationsPage.goToUsers();
+	await (
+		await usersAndOrganizationsPage.usersTableRowActions(
+			`${user.alternateName}`
+		)
+	).click();
+
+	const pagePromise = context.waitForEvent('page');
+	await usersAndOrganizationsPage.impersonateUserMenuItem.click();
+
+	const newPage = await pagePromise;
+	await newPage.getByTestId('userPersonalMenu').click();
```

Comment:
@victorcdp do you think is it possible to do something like this ?
```
const xxx = new ***Page(newPage);
```
so we can have the locators inside the pages?